samples/stock/xaxis/overscroll-percent/demo.css

```diff
@@ -0,0 +1,4 @@
+#container {
+    height: 400px;
+    min-width: 600px;
```

Comment:
Unnecessary restriction, I confirmed it works well on the mobile. Set `min-width: 320px` as usual.